README.md

```diff
@@ -1,17 +1,18 @@
 
 # masterCaptain
+
 ## MVP Presentation
 
-# Legacy
+  ## Who is your target audience?
+   * Our target audience is volunteers of different demographic groups who are willing to change the World and want be a part of large Non-Profit Organizations which are making the difference.
+   
+   ## The goals of the product
+ * To match people interests with the volunteer Organizations
+ * Increase # of volunteers
+ * Provide Volunteers with useful resource to plan a Volunteer Project
+
+
+ ## What is the problem that the product will address
+  * “Legacy” will direct our customers and simplify the path of planning a Volunteer Project in order to safe time, energy and money and connect our customers with the right resources ( in our case with Non-Profit organizations).
```

Comment:
We should change the name from "Legacy" to something else.

Reply:
sure thing